src/main/java/com/gooddata/authentication/SstAuthentication.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (C) 2004-2016, GoodData(R) Corporation. All rights reserved.
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE.txt file in the root directory of this source tree.
+ */
+package com.gooddata.authentication;
+
+import com.gooddata.GoodDataEndpoint;
+import com.gooddata.Authentication;
+import com.gooddata.http.client.GoodDataHttpClient;
+import com.gooddata.http.client.SSTRetrievalStrategy;
+import com.gooddata.http.client.SimpleSSTRetrievalStrategy;
+import org.apache.http.client.HttpClient;
+import org.apache.http.impl.client.HttpClientBuilder;
+
+import static com.gooddata.util.Validate.notEmpty;
+import static com.gooddata.util.Validate.notNull;
+
+/**
+ * SST authentication
```

Comment:
Let's be a bit more descriptive - average user from public has no idea what SST means. May be a link to developer portal would be also helpful?